src/main/scala/is/hail/HailContext.scala

```diff
@@ -471,7 +476,8 @@ class HailContext private(val sc: SparkContext,
     path: String,
     partFiles: Array[String],
     read: (Int, InputStream, InputMetrics) => Iterator[T],
-    optPartitioner: Option[Partitioner] = None): RDD[T] = {
+    optPartitioner: Option[Partitioner] = None
+  ): RDD[T] = {
```

Comment:
I'll  say again, you're not doing yourself (or the reviewers) any favors if you gratuitously change the formatting.